src/linux/net/wifi/core/AccessPointControllerLinux.cxx

```diff
@@ -394,6 +504,42 @@ AccessPointControllerLinux::SetPairwiseCipherSuites(std::unordered_map<Ieee80211
         return status;
     }
 
+    // Validate that the pairwise cipher suites were successfully set by hostapd.
+    auto hostapdBssConfiguration = m_hostapd.GetConfiguration();
+    auto wpaSecurityProtocol = hostapdBssConfiguration.Wpa;
+    bool isWpaSecurityProtocolSet = std::ranges::contains(pairwiseCipherSuitesHostapd, wpaSecurityProtocol, [](const auto& wpaCipher) {
+        return wpaCipher.first;
+    });
+    if (isWpaSecurityProtocolSet) {
+        AUDITI << std::format("WPA security protocol for AP {} set to {}", status.AccessPointId, std::to_underlying(wpaSecurityProtocol));
+    } else {
+        AUDITD << std::format("WPA security protocol for AP {} was not set. Current wpa value: {}", status.AccessPointId, std::to_underlying(wpaSecurityProtocol));
```

Comment:
Using `std::to_underlying` here will put numerical enum value in the log. Wouldn't it be better to have the name instead, using the `*ToString()` function or `magic_enum::enum_name(wpaSecurityProtocol)`? Same for instances of `std::to_underlying` used below.

Reply:
Yeah, the name is probably better. My reasoning for using the numerical value was that it matches what gets returned/displayed by the hostapd CLI "GET_CONFIG" command (e.g. "wpa=2").